### PR TITLE
add: checkout page

### PR DESCRIPTION
**Pull request by saurabhchirde**

### CODE REVIEW

**Review by 05him — APPROVED** [review]
Your UI is awesome and code is clean too 
Its Good To GO

**Review by abhijeetekad — APPROVED** [review]

File: src/style.css [hunk 1]
```diff
@@ -25,6 +26,12 @@ body,
     color: var(--sub-title-color);
 }
 
+.desktop-navigation-bar .avatar,
+.mobile-navigation-bar .avatar {
+    margin: 0 0.6rem;
```
Comment (zoyatanweer) [inline src/style.css:31]: 0.6rem can be in px

File: src/style.css [hunk 2]
```diff
@@ -349,22 +356,30 @@ form>div>button {
...
     font-size: 1rem;
 }
 
 .price-table-dark h3 {
     color: var(--sub-title-color);
 }
 
+.cart-price-table .cart-btns a {
+    display: grid;
+    margin: 0.4rem 0;
```
Comment (zoyatanweer) [inline src/style.css:375]: this can be in px

File: src/style.css [hunk 3]
```diff
@@ -373,19 +388,152 @@ form>div>button {
...
+.cart-products,
+.checkout-form {
     width: 100%;
     flex-direction: column;
     align-items: center;
     justify-content: center;
 }
 
+.checkout-form>div {
+    margin: 0 0.8rem 0.8rem 0.8rem;
```
Comment (zoyatanweer) [inline src/style.css:400]: these can be in px

File: src/style.css [hunk 4]
```diff
@@ -373,19 +388,152 @@ form>div>button {
...
+
+.contact-address>details>button,
+.payment-method>details>button,
+.checkout-item-overview>details>button {
+    margin-left: 2.2rem;
+}
+
+.address {
+    width: 100%;
+    margin: 0.4rem 0;
```
Comment (zoyatanweer) [inline src/style.css:463]: in px

File: src/style.css [hunk 5]
```diff
@@ -373,19 +388,152 @@ form>div>button {
...
+    height: auto;
+    border-radius: 4px;
+    padding: 1rem 0.4rem 0.4rem;
```
Comment (zoyatanweer) [inline src/style.css:466]: in px

File: src/style.css [hunk 6]
```diff
@@ -373,19 +388,152 @@ form>div>button {
...
+    margin-top: 1.5rem;
+}
+
+.checkout-price-table {
+    width: 80rem;
+    margin-top: 0;
+}
+
+.checkout-price-table>div {
+    padding: 0.5rem 0;
```
Comment (zoyatanweer) [inline src/style.css:523]: in px

File: src/style.css [hunk 7]
```diff
@@ -894,6 +1059,14 @@ h3 {
         margin-right: 0.2rem;
```
Comment (zoyatanweer) [inline src/style.css:1059]: in px

File: src/style.css [hunk 8]
```diff
@@ -856,33 +1015,39 @@ h3 {
...
     }
 
-    .cart-item-card .card-text h1 {
+    .cart-item-card .card-text h1,
+    .checkout-item-card .card-text h1 {
         width: 100%;
         font-size: 0.8rem;
         margin: 0;
         margin-top: 0.6rem;
     }
```
Comment (zoyatanweer) [inline src/style.css:1047]: write in px

**Review by zoyatanweer — CHANGES_REQUESTED** [review]
UI is awesome! clean code and after suggested changes you are good to merge

**Review by zoyatanweer — APPROVED** [review]

File: src/style.css (hunk 1 above)
Comment (saurabhchirde) [reply src/style.css:31]: I am following the practice to use px only for the border radius, to make it appear the same on all font sizes of the root.

**Pull request merged**